clusterloader2/pkg/measurement/common/kube_state_metrics_measurement.go

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+
+	"github.com/prometheus/common/model"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
```

Comment:
nit: can you please sort the non-builtin imports alphabetically?

Reply:
I can try, but vim-go sorts it automatically this way for me.

Reply:
Update: I ran both gofmt as well golang.org/x/tools/cmd/goimports and both did not change the ordering here. If we switch the ordering, next time someone with gofmt plugin like mine modifies we will have changes.

Curious to know what do you use?

Reply:
gofmt doesn't change order of important (I don't know about goimports).

I'm just using my eyes :)

Reply:
From what I can tell, it orders it by org, then repo, think that makes sense here. vim-go has both goftm and goimports.

https://goinbigdata.com/goimports-vs-gofmt/